requirements-freqai.txt

```diff
@@ -1,9 +1,14 @@
 # Include all requirements to run the bot.
--r requirements.txt
+-r requirements-hyperopt.txt
```

Comment:
All requirements from hyperopt will be needed for freqAI now?

Reply:
No they shouldn't be - this line seems to be left over.